frontend/src/components/IssueSidebar.vue

```diff
@@ -313,6 +319,15 @@ export default defineComponent({
         : null,
     });
 
+    watch(
+      () => props.task,
```

Comment:
Missed this one before. Let's NOT use props.task here, instead let the Caller IssueDetail derive the earliestAllowedTs and pass that. See how we pass database/instance to IssueSidebar. Couple reasons:

1. IssueDetail is the central place, where we may pass earliestAllowedTs to other sub component as well.
2. Be consistent with how we pass database/instance and we handle the updateEarliestAllowedTime in IssueDetail (BTW, updateEarliestAllowedTime has a leftover console.log))

Reply:
But we still need a local state for the n-date-picker to bind its value. And if we bind such a props value, it would cause a non-mutation change, which is prohibited by Vue

Reply:
Fair enough, then it's OK as is as it's different from the instance/database usage.

On the other hand, the template should use state.updateEarliestAllowedTime instead of task. updateEarliestAllowedTime ?